contracts/adapters/voting/Voting.sol

```diff
@@ -61,13 +62,32 @@ contract VotingContract is IVoting, DaoConstants, MemberGuard, AdapterGuard {
     function startNewVotingForProposal(
         DaoRegistry dao,
         uint256 _proposalId,
-        bytes calldata
+        bytes memory data
```

Comment:
why did you need to change from calldata -> memory ?

Reply:
To correct a compiler error.